Content.Server/GameObjects/Components/Chemistry/MovespeedModifierMetabolismComponent.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Shared.Movement.Components;
+using Robust.Shared.GameObjects;
+using Robust.Shared.ViewVariables;
+using System;
+using System.Threading;
+
+namespace Content.Server.GameObjects.Components.Chemistry
+{
+    //TODO: refactor movement modifier component because this is a pretty poor solution
+    [RegisterComponent]
+    class MovespeedModifierMetabolismComponent : Component, IMoveSpeedModifier
+    {
+        
+        public override string Name => "ChemicalMovementSpeedChangeStatus";
```

Comment:
component name should line up with the class name, i.e.

```suggestion
        public override string Name => "MovespeedModifierMetabolism";
```